src/app/search/search.component.ts

```diff
@@ -261,6 +265,12 @@ export class SearchComponent implements OnInit, OnDestroy {
     });
   }
 
+  getHistogramWidth(items: number) {
+    const rowWidth = document.getElementById('facet-row').offsetWidth;
+    const histogramWidth = (items / 1000) * rowWidth;
```

Comment:
Where does 1000 come from?

Reply:
This gets the number of items to become a decimal to calculate the width of the histogram. It probably shouldn't be hard coded as 1000, though I'm not sure how else we can evaluate the number of items as a percentage of something